doc/DESI_SPECTRO_REDUX/SPECPROD/exposures/NIGHT/EXPID/qframe-CAMERA-EXPID.rst

```diff
@@ -0,0 +1,225 @@
+BZERO    2147483648       int
+CHECKSUM gaU7iZS4gaS4gWS4 str  HDU checksum updated 2016-06-10T16:57:58
+DATASUM  737750           str  data unit checksum updated 2016-06-10T16:57:58
+======== ================ ==== ==============================================
+
+Data: FITS image [int32 (compressed), 2951x500]
+
+HDU3
+----
+
+EXTNAME = SIGMA
+
+LSF sigma[nspec, nwave], in CCD pixel units. 
+
+Required Header Keywords
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+======== ================ ==== ==============================================
+KEY      Example Value    Type Comment
+======== ================ ==== ==============================================
+NAXIS1   2951             int  length of original image axis
+NAXIS2   500              int  length of original image axis
+BSCALE   1                int
+BZERO    2147483648       int
+CHECKSUM gaU7iZS4gaS4gWS4 str  HDU checksum updated 2016-06-10T16:57:58
+DATASUM  737750           str  data unit checksum updated 2016-06-10T16:57:58
+======== ================ ==== ==============================================
+
+Data: FITS image [float32]
+
+HDU4
+----
+
+EXTNAME = WAVELENGTH
+
+2D array of wavelengths[nspec, nwave]
+
+Required Header Keywords
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+======== ================ ==== =====================
+KEY      Example Value    Type Comment
+======== ================ ==== =====================
+NAXIS1   2951             int  length of original image axis
+NAXIS2   500              int  length of original image axis
+BUNIT    Angstrom         str
+CHECKSUM gaU7iZS4gaS4gWS4 str  HDU checksum updated 2016-06-10T16:57:58
+DATASUM  737750           str  data unit checksum updated 2016-06-10T16:57:58
+======== ================ ==== =====================
+
+Data: FITS image [float64]
+
+HDU5
+----
+
+EXTNAME = FIBERMAP
+
+Fibermap propagated from the raw data inputs; see
+:doc:`fibermap file <../../../../../DESI_SPECTRO_DATA/NIGHT/EXPID/fibermap-EXPID>`.
+
+Required Header Keywords
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+======== ================ ==== ==============================================
+KEY      Example Value    Type Comment
+======== ================ ==== ==============================================
+NAXIS1   378              int  length of dimension 1
+NAXIS2   500              int  length of dimension 2
+ENCODING ascii            str
+CHECKSUM UUVAVUS8UUSAUUS5 str  HDU checksum updated 2018-03-01T15:08:15
+DATASUM  4154192770       str  data unit checksum updated 2018-03-01T15:08:15
+======== ================ ==== ==============================================
+
+Required Data Table Columns
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+=========== ========== ===== ===========
+Name        Type       Units Description
+=========== ========== ===== ===========
+OBJTYPE     char[10]
+TARGETCAT   char[20]
+BRICKNAME   char[8]
+TARGETID    int64
+DESI_TARGET int64
+BGS_TARGET  int64
+MWS_TARGET  int64
+MAG         float32[5]
+FILTER      char[200]
+SPECTROID   int64
+POSITIONER  int32
```

Comment:
At least some of the columns in this table do have physical units, so those should be in the Units column.